nbs/core.ipynb

```diff
@@ -35,7 +35,7 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #222.            df : pandas, polars or spark DataFrame, or a list of files containing the DataFrame, optional (default=None)
</pre><p>Please change this to: <code>pandas, polars or spark DataFrame, or a list of parquet files containing the series, optional (default=None)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>